code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -300,19 +300,24 @@ Des: Removes all infected images from the alien.
 	return
 
 /mob/living/carbon/alien/proc/can_evolve()
+	var/list/alive_aliens = aliens
```

Comment:
aliens is a list. lists are objects. if you do var/list/mylist = a_global_list, editing mylist will also edit a_global_list.
to fix, replace aliens with aliens.Copy()